internal_metrics/dimensions.yaml

```diff
@@ -0,0 +1,171 @@
+  example: 5.4.219-126.411.amzn2.x86_64 
+
+k8s.cluster.name:
+  brief: Kubernets Cluster name.
+  description: Kubernetes Cluster name.
+  associated_metrics: ebpf_net.up, ebpf_net.time_since_last_message_ns, ebpf_net.span_utilization, ebpf_net.span_utilization_fraction, ebpf_net.span_utilization_max, ebpf_net.rpc_queue_elem_utilization_fraction, ebpf_net.rpc_queue_buf_utilization, ebpf_net.rpc_queue_buf_utilization_fraction, ebpf_net.rpc_latency_ns, ebpf_net.pipeline_metric_bytes_discarded, ebpf_net.pipeline_metric_bytes_written, ebpf_net.pipeline_message_error, ebpf_net.otlp_grpc.unknown_response_tags, ebpf_net.otlp_grpc.requests_sent, ebpf_net.otlp_grpc.bytes_sent, ebpf_net.otlp_grpc.bytes_failed, ebpf_net.message, ebpf_net.entrypoint_info, ebpf_net.connections, ebpf_net.disconnects, ebpf_net.collector_log_count, ebpf_net.collector_health, ebpf_net.bpf_log, ebpf_net.agg_root_truncation, ebpf_net.client_handle_pool, ebpf_net.client_handle_pool_fraction, ebpf_net.codetiming_min_ns, ebpf_net.codetiming_max_ns, ebpf_net.codetiming_sum_ns, ebpf_net.codetiming_avg_ns, ebpf_net.codetiming_count
+  example: staging.
+
+line:
+  brief: Line number of the code block instrumented.
+  description: Line number of the code block instrumented..
+  associated_metrics: ebpf_net.bpf_log, ebpf_net.codetiming_min_ns, ebpf_net.codetiming_max_ns, ebpf_net.codetiming_sum_ns, ebpf_net.codetiming_avg_ns, ebpf_net.codetiming_count
+  example: 170
+
+message:
+  brief: Message type
+  description: Message type extracted by collector at the linux kernel level.
+  associated_metrics: ebpf_net.message, ebpf_net.pipeline_message_error
+  example: task_info, set_cgroup, pid_close_info
+
+module:
+  brief: Name of the Reducer core
+  description: Name of the Reducer core.
+  associated_metrics: ebpf_net.up, ebpf_net.span_utilization, ebpf_net.span_utilization_fraction, ebpf_net.span_utilization_max, ebpf_net.rpc_queue_elem_utilization_fraction, ebpf_net.rpc_queue_buf_utilization, ebpf_net.rpc_queue_buf_utilization_fraction, ebpf_net.rpc_latency_ns, ebpf_net.pipeline_metric_bytes_discarded, ebpf_net.pipeline_metric_bytes_written, ebpf_net.pipeline_message_error, ebpf_net.otlp_grpc.unknown_response_tags, ebpf_net.otlp_grpc.requests_sent, ebpf_net.otlp_grpc.bytes_sent, ebpf_net.otlp_grpc.bytes_failed, ebpf_net.message, ebpf_net.entrypoint_info, ebpf_net.connections, ebpf_net.disconnects, ebpf_net.collector_log_count, ebpf_net.bpf_log, ebpf_net.agg_root_truncation, ebpf_net.client_handle_pool, ebpf_net.client_handle_pool_fraction, ebpf_net.codetiming_min_ns, ebpf_net.codetiming_max_ns, ebpf_net.codetiming_sum_ns, ebpf_net.codetiming_avg_ns
+  example: ingest
+
+name:
+  brief: Name of the instrumented codeblock.
+  description: Name of the instrumented codeblock.
+  associated_metrics: ebpf_net.bpf_log, ebpf_net.codetiming_min_ns, ebpf_net.codetiming_max_ns, ebpf_net.codetiming_sum_ns, ebpf_net.codetiming_avg_ns, ebpf_net.codetiming_count
```

Comment:
ebpf_net.codetiming* may also be developer/debug metrics that we shouldn't document?